src/components/calculator.js

```diff
@@ -1,39 +1,51 @@
+          <button className="btn" type="button" onClick={() => handleButtonClick('+/-')}>+/-</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('%')}>%</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('+')}>+</button>
         </div>
         <div className="row">
-          <button className="btn" type="button">7</button>
-          <button className="btn" type="button">8</button>
-          <button className="btn" type="button">9</button>
-          <button className="btn" type="button">*</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('7')}>7</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('8')}>8</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('9')}>9</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('x')}>*</button>
         </div>
         <div className="row">
-          <button className="btn" type="button">4</button>
-          <button className="btn" type="button">5</button>
-          <button className="btn" type="button">6</button>
-          <button className="btn" type="button">-</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('4')}>4</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('5')}>5</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('6')}>6</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('-')}>-</button>
         </div>
         <div className="row">
-          <button className="btn" type="button">1</button>
-          <button className="btn" type="button">2</button>
-          <button className="btn" type="button">3</button>
-          <button className="btn" type="button">+</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('1')}>1</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('2')}>2</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('3')}>3</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('+')}>+</button>
         </div>
         <div className="row">
-          <button className="btn zero" type="button">0</button>
-          <button className="btn" type="button">.</button>
-          <button className="btn" type="button">=</button>
+          <button className="btn zero" type="button" onClick={() => handleButtonClick('0')}>0</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('.')}>.</button>
+          <button className="btn" type="button" onClick={() => handleButtonClick('=')}>=</button>
         </div>
       </div>
     </div>
```

Comment:
- [ ] [REQUIRED] Kindly add the division to your calculator, currently, it has double addition, let;s remove one and add the division where expected.
<img width="475" alt="Screenshot 2023-08-12 at 00 21 20" src="https://github.com/mohamedgamaleldin999999/Math-Magicians/assets/58692590/0ef783a6-cd7e-4fe2-90ee-e2e5f1002a29">